packages/zapp/console/src/components/Launch/LaunchForm/constants.ts

```diff
@@ -4,13 +4,17 @@ import { BlobValue, InputType, AuthRoleTypes, AuthRoleMeta } from './types';
 export const formStrings = {
   cancel: 'Cancel',
   inputs: 'Inputs',
+  gateInput: 'Gate input',
   role: 'Role',
   submit: 'Launch',
+  resume: 'Resume',
   taskVersion: 'Task Version',
   title: 'Create New Execution',
+  resumeTitle: 'Resume Paused Execution',
   workflowVersion: 'Workflow Version',
   launchPlan: 'Launch Plan',
   interruptible: 'Interruptible',
+  viewNodeInputs: 'View node inputs',
 };
```

Comment:
let's move all of the formStrings and lines 77-90 (except line 83 `workflowNoInputsString`, as it's being imported outside of the LaunchForm too) in a localization strings file for this folder